etc/infrastructure/php/php.ini-production

```diff
@@ -0,0 +1,1932 @@
+[PHP]
```

Comment:
Why would we need a `php.ini` configuration for the `-development` environment different than the `-production` one?

We should only have one single `php.ini` file. This way we would be sure that we're running the same configuration in development and production, avoiding possible production bugs not reproducible in the  development environment :)

Reply:
Yes, nice point! this is copied from an example and put maybe without much thinking, btw this files aren't used in the docker-compose files, so I think it's better to remove at the moment both files, and if it's necessary in a future add